ovs/extensions/storage/persistent/arakoonstore.py

```diff
@@ -123,3 +134,19 @@ def _try(method, *args, **kwargs):
                 tries -= 1
                 time.sleep(1)
         raise last_exception
+
+    @staticmethod
+    def _next_key(key):
+        """
+        Calculates the next key (to be used in range queries)
+        """
+        encoding = 'ascii'  # For future python 3 compatibility
+        array = bytearray(str(key), encoding)
+        for index in range(len(array) - 1, -1, -1):
+            array[index] += 1
+            if array[index] < 128:
+                while array[-1] == 0:
+                    array = array[:-1]
+                return str(array.decode(encoding))
+            array[index] = 0
+        return '\xff'
```

Comment:
so you're claiming '\xff' is the biggest possible key?
(btw for this sort of methods unit tests can be handy...)